ui.apps/src/main/content/jcr_root/apps/etoolbox-backpack/admin/console/page/content/clientlibs/js/paths.js

```diff
@@ -14,8 +14,40 @@
 $(function () {
     "use strict";
 
+    $(document).on("click", '.js-create-package-activator', function(event) {
+        var dialog = document.querySelector('#createPackageDialog');
+        if (!dialog) {
+            var path = $('.js-create-package-activator').data('path');
+            $.ajax({
+                type: "GET",
+                url: '/apps/etoolbox-backpack/admin/console/page/content/createpagedialog.html?paths=' + path,
+                success: function(data) {
+                    var doc = new DOMParser().parseFromString(data, 'text/html');
+                    document.body.appendChild(doc.body.firstChild);
+                }
+            });
+        }
+        setTimeout(function() {
+            dialog = document.querySelector('#createPackageDialog');
+            fillPaths();
+            dialog.show();
+        }, 300);
+    });
+
     $(document).on("foundation-contentloaded", function() {
 
+        fillPaths();
+
+        var switchField = $('#create-package-switch')[0];
```

Comment:
Look, you do a contradictory thing:
1) init a JQuery object with `$('#create-package-switch')`
2) then de-structure it into a DOM object with `[0]`
3) then re-init all the same JQuery object with `$(switchField)`

Just do `$('#create-package-switch').on('change'...`